### PR TITLE
include instructions for how to be added to the `hug bomb`

### PR DESCRIPTION
**Pull request by afeld**
Also made it easier to add new images: no environment variables need be changed, nor do the images need any special naming convention.

/cc @dlapiduz

### CODE REVIEW

File: scripts/hugme.coffee [hunk 1]
```diff
@@ -12,28 +12,30 @@
...
+  s3.listObjects {Bucket: BUCKET}, (err, data) ->
+    if err
+      msg.reply("Error retrieving images: #{err}")
+    else
+      # send unique URLs
+      s3Objects = _u.sample(data.Contents, count)
+      for s3Object in s3Objects
+        url = hugUrl(s3Object)
+        msg.reply(url)
+      msg.reply("_If you would like to be added, send a picture in #bots._")
```
Comment (afeld) [inline scripts/hugme.coffee]: This is all I _really_ wanted to add, but got carried away :grin:

**Comment by dlapiduz** [comment]
This is going to send a message on every hug bomb, right? do we really need that?

**Comment by afeld** [comment]
I feel like the question comes up almost every time `hug bomb` is used...open to other ideas of where to include instructions.

**Comment by dlapiduz** [comment]
fair point. Ok lets try it...

**Pull request merged**